roles/ceph-common/tasks/installs/install_on_redhat.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: install dependencies
+- name: install redhat dependencies yum
```

Comment:
perhaps we can stay consistent with your last change and keep using "via yum"

same goes for the next tasks.